test/core/end2end/tests/simple_request.cc

```diff
@@ -225,17 +225,18 @@ static void simple_request_body(grpc_end2end_test_config config,
 
   cq_verifier_destroy(cqv);
 
+  int expected_calls = 1;
+  if (config.feature_mask & FEATURE_MASK_SUPPORTS_REQUEST_PROXYING) {
+    expected_calls *= 2;
+  }
+#if defined(GRPC_COLLECT_STATS) || !defined(NDEBUG)
+
   grpc_stats_collect(after);
```

Comment:
Earlier in the file, we have `grpc_stats_collect(before)` running despite whether GRPC_COLLECT_STATS is on or not. Could you move that inside an `#if defined(GRPC_COLLECT_STATS) || !defined(NDEBUG)` as well for consistency?

Reply:
Done